src/main/telemetry/crsf.c

```diff
@@ -112,7 +112,7 @@ bool handleCrsfMspFrameBuffer(uint8_t payloadSize, mspResponseFnPtr responseFn)
     int pos = 0;
     while (true) {
         const int mspFrameLength = mspRxBuffer.bytes[pos];
-        if (handleMspFrame(&mspRxBuffer.bytes[CRSF_MSP_LENGTH_OFFSET + pos], mspFrameLength)) {
+        if (handleMspFrame(&mspRxBuffer.bytes[CRSF_MSP_LENGTH_OFFSET + pos], mspFrameLength, NULL)) {
```

Comment:
This will make `handleMspFrame()` write to `NULL`.

Reply:
What's the best way to deal with it? Can i do ``if (packet->cmd == MSP_EEPROM_WRITE && skipsBeforeResponse)``? I'm unsure.

Reply:
Yeah, that will do. Or send a dummy variable in `crsf.c`. Both are valid ways to address this.

Reply:
Thanks! Fixes are coming